async-clipboard/index.html

```diff
@@ -0,0 +1,23 @@
+---
+feature_name: Asynchronous Clipboard API
+chrome_version: 66
+feature_id: 5861289330999296
+local_css_files: ['demo.css']
+additional_head_content: <meta name="optional" value="optional">
```

Comment:
Newbee question: What is it for?

Reply:
whoop - it's from the template, forgot to remove it.